src/structures/MessageEmbed.js

```diff
@@ -242,7 +242,8 @@ class MessageEmbed {
       (this.fields.length >= 1
         ? this.fields.reduce((prev, curr) => prev + curr.name.length + curr.value.length, 0)
         : 0) +
-      (this.footer ? this.footer.text.length : 0)
+      (this.footer?.text.length ?? 0) +
+      (this.author?.name.length ?? 0)
```

Comment:
i think you should use the tenary operator here as before, makes more sense to me IMO (also this makes it inconsistent with how `this.title` and `this.description` length is retrieved here)

Reply:
So the other 2 should be changed.